objects/my_object.rb

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env ruby
+
+# Check to see if your coffee maker is ready with .ready?
+# See what you're missing with .display_errors
+# Add water, filter, or coffee_grounds with .add_water(how_much?) (etc)
+# Brew your coffee with .brew
+# If you try to brew when it's not ready it will tell you that it's not ready
+class CoffeeMaker
+  def initialize
+    @water = 0
+    @grounds = 0
+    @filter = false
+  end
+
+  def add_water(number_of_cups)
+    @water = number_of_cups
+  end
+
+  def add_filter
+    @filter = true
+  end
+
+  def add_coffee_grounds(amount)
+    @grounds = amount
+  end
+
+  def ready?
+    if water? && filter? && coffee_grounds?
```

Comment:
You don't need the if here...  This whole method could be `water? && filter? && coffee_grounds?`.